dotnet3.5/S3/CreateBucketExample/CreateBucket/CreateBucket.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+
+// snippet-start:[S3.dotnet35.CreateBucket]
+
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using Amazon.S3.Util;
+using System;
+using System.Threading.Tasks;
+
+namespace CreateBucket
+{
+    public class CreateBucket
+    {
+        // This example shows how to use Amazon Simple Storage Service (Amazon S3)
+        // to create a new Amazon S3 bucket. The examples uses AWS SDK for .NET 3.5 and
+        // .NET 5.0.
+
+        // Specify your AWS Region (an example Region is shown).
+        private static readonly RegionEndpoint BUCKET_REGION = RegionEndpoint.USWest2;
+        private static IAmazonS3 _s3Client;
+
+        // Specify the name of the new bucket.
+        private const string NEW_BUCKET_NAME = "doc-example-bucket";
+
+        static async Task Main()
+        {
+            _s3Client = new AmazonS3Client(BUCKET_REGION);
+            Console.WriteLine($"\nCreating a new bucket, named: {NEW_BUCKET_NAME}.");
+
+            await CreatingBucketAsync(_s3Client, NEW_BUCKET_NAME);
+
+            var bucketLocation = FindBucketLocationAsync(_s3Client, NEW_BUCKET_NAME);
+
+            if(!String.IsNullOrEmpty(bucketLocation.ToString()))
+            {
+                Console.WriteLine($"\n\nBucket {NEW_BUCKET_NAME} successfully created.");
+            }
+            else
+            {
+                Console.WriteLine("The bucket does not exist.");
```

Comment:
One possible edge case here - I believe the bucket's location is null for `us-east-1` - https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetBucketLocation.html#API_GetBucketLocation_ResponseElements

Fine in this specific example since we hardcoded us-west-2 above, but don't want to lead someone astray if they assume `IsNullOrEmpty` always means `The bucket does not exist`

Reply:
Removed the test for BucketLocation as it isn't really necessary in this example.

Reply:
I removed the code to get the location. It isn't necessary and only confuses the issue at hand which is creating a bucket.